playwright-e2e/package-lock.json

```diff
@@ -31,6 +30,9 @@
         "@typescript-eslint/parser": "^5.54.0",
         "eslint": "^8.35.0",
         "eslint-plugin-import": "^2.27.5",
+        "eslint-plugin-prettier": "^4.2.1",
+        "prettier": "^2.8.4",
```

Comment:
I have removed prettier and related plugin. Do you think we want to keep it?

Reply:
no - let's take it out (will do!) - it ended up getting put in while using vs code